### DISCUSSION TITLE
All plugins don't work [BTCPay Server Version v2.0.5+6ae36825d]

### DISCUSSION THREAD

**Discussion by juniorcines** [btcpayserver/btcpayserver#6523] [Plugins & integrations]
Hello, since I updated to version v2.0.5+6ae36825d

all plugins are not working for me.

no matter which plugin i install, they all don't work.

**Comment by pavlenex** [comment]
When reporting an issue, it would be great if  you could first name list of plugins and then if there's anything in server logs as well, otherwise it'll be very hard to provide support.
